evalml/automl/engine/engine_base.py

```diff
@@ -304,26 +328,105 @@ def train_and_score_pipeline(
         ):
             evaluation_entry["binary_classification_threshold"] = cv_pipeline.threshold
         cv_data.append(evaluation_entry)
-    training_time = time.time() - start
     cv_scores = pd.Series([fold["mean_cv_score"] for fold in cv_data])
     cv_score_mean = cv_scores.mean()
     logger.info(
         f"\tFinished cross validation - mean {automl_config.objective.name}: {cv_score_mean:.3f}"
     )
+
+    holdout_score = np.NaN
+    holdout_scores = np.NaN
+    if X_holdout is not None and y_holdout is not None:
+        logger.info("\tStarting holdout set scoring")
```

Comment:
Yep, it prints for every single pipeline. I figured that since we're printing out the CV scores, it would be useful to print out the holdout scores as well. This is especially true since we're using this score as part of the search iteration plot. 

![image](https://user-images.githubusercontent.com/8752455/178800437-25d118e2-2107-463e-bb4e-da5760ff294b.png)

Reply:
gotcha - looks way better than I expected. Keep it in!